build/version.settings.targets

```diff
@@ -2,7 +2,7 @@
   <PropertyGroup>    
     <!--SxS: These three properties should be changed at the start of a new version, VersionZeroYear should be the year
     before the start of the project. When updating the version, also update MIEngine\metadata.json.-->
-    <MajorVersion>14</MajorVersion>
+    <MajorVersion>15</MajorVersion>
     <MinorVersion>0</MinorVersion>
     <VersionZeroYear>2013</VersionZeroYear>
```

Comment:
It says what real year should have a '1' in the year part of the version number (MIEngine uses build numbers of YMMDD). But the max number of years which this scheme can accept is 6, so when we rev the major year, we also want to update the VersionZeroYear.